docs/spec/relayer/Relayer.tla

```diff
@@ -233,25 +161,34 @@ ChannelDatagrams(srcChainID, dstChainID) ==
     ELSE {}
     
 (***************************************************************************
- Compute pending datagrams (from srcChainID to dstChainID)
+ Compute datagrams (from srcChainID to dstChainID)
  ***************************************************************************)
 \* Currently supporting:
 \*  -  ICS 002: Client updates
 \*  -  ICS 003: Connection handshake
 \*  -  ICS 004: Channel handshake
 \* TODO: Support the remaining datagrams
-PendingDatagrams(srcChainID, dstChainID) ==
+ComputeDatagrams(srcChainID, dstChainID) ==
     \* ICS 002 : Clients
     \* - Determine if light clients needs to be updated 
-    LET clientDatagrams == LightClientUpdates(srcChainID, dstChainID, relayerChainHeights) IN
+    LET clientDatagrams == 
+        IF GenerateClientDatagrams 
+        THEN LightClientUpdates(srcChainID, dstChainID, relayerHeights) 
```

Comment:
Where relayerHeights come from?I guess idea here is to update relayer's height for chainA and chainB even if we don't generate clientDatagrams?

Reply:
The `LightClientUpdates` may trigger the relayer to update its client heights, in case they are not up to date with the height of the chain involved in the creation of client datagrams. When the relayer does not create client datagrams, these we don't update the relayer's client heights, that is, we output the heights that were given as input (by setting `relayerUpdate |-> relayerHeights`).